test/format/config_def.c

```diff
@@ -0,0 +1,357 @@
+/* DO NOT EDIT: automatically built by format/config.sh. */
+
+#include "format.h"
+
+CONFIG configuration_list[] = {
```

Comment:
We can remove this file from the PR since it seems to be replaced with `format_config_def.c`

Reply:
yup! it has been fixed here [0f3ef44](https://github.com/wiredtiger/wiredtiger/pull/8608/commits/0f3ef4417c4a2daa82c2f38c85fc965d13329273)